src/components/Molecules/Menu/index.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <nav>
+    <div class="menuIcon">
+      <div class="menuLogo">LOGO</div>
+      <i class="fas fa-bars"></i>
+      <Awesome icon="bars" class="hamburgerMenu" @click="menuToggle()" />
+    </div>
+    <div class="displayNoneMenu" id="menuToggle">
+      <div class="menuLogo hide">LOGO</div>
+      <ul class="menuItem">
+        <a href="#">
+          <li>Strona główna</li>
+        </a>
+        <a href="#">
+          <li>O nas</li>
+        </a>
+        <a href="#">
+          <li>Jak to działa</li>
+        </a>
+        <a href="#">
+          <li>Cennik</li>
+        </a>
+        <a href="#">
+          <li>Kontakt</li>
+        </a>
+      </ul>
+      <div class="menuButtonBox">
+        <a href>Zarejestruj się</a>
+        <span class>lub</span>
+        <Button value="Zaloguj się" class="small" />
```

Comment:
nice, but I wonder if that signature will be possible
```<Button class="small">Zaloguj się</Button>```

Reply:
Slots will be added in the coming days